dcos/mesos.py

```diff
@@ -496,7 +496,7 @@ def _get_container_status(task):
         def _get_container_id(container_status):
             if 'container_id' in container_status:
                 if 'value' in container_status['container_id']:
-                    return container_status['container_id']['value']
+                    return container_status['container_id']
```

Comment:
Reverting this breaks task metrics (`dcos task metrics details|summary <task-id>`) can we refactor a little?